include/sys/mm.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_SYS_MM_H
```

Comment:
`mm` is a little opaque; could this be `<sys/memmap.h>` instead? Or `<sys/memmgmt.h>` if that's a better name?

Reply:
I'd vote for memmap.h

Reply:
I renamed it to `mem_manage.h`.